resources/views/admin/api_keys.php

```diff
@@ -29,12 +29,12 @@
             <tbody>
             <tr>
                 <th scope="row">
-                     <label for="app-key">App Key*</label>
+                    <label for="app-key">App Key*</label>
                 </th>
                 <td>
                     <input type="text" name="app_key" id="app-key" class="regular-text"
-                            value="<?php echo $options['app_key']; ?>">
-                    <?php echo $getvalidation_icon('app_key')?>
+                           value="<?php echo $options['app_key']; ?>">
```

Comment:
@Osoro Shouldn't we validate `$options` from [resourcers/controllers/Admin.php](https://github.com/zype/zype-wordpress-plugin/pull/38/files#diff-79dfb295029a45fe9eca518c86fbb08eR85)?